src/DynamoCoreWpf/PublicAPI.Unshipped.txt

```diff
@@ -2748,6 +2748,8 @@ Dynamo.ViewModels.PortViewModel.ConnectCommand.get -> Dynamo.UI.Commands.Delegat
 Dynamo.ViewModels.PortViewModel.EventType.get -> Dynamo.Graph.Nodes.PortEventType
 Dynamo.ViewModels.PortViewModel.EventType.set -> void
 Dynamo.ViewModels.PortViewModel.Height.get -> double
+Dynamo.ViewModels.PortViewModel.Highlight.get -> System.Windows.Visibility
```

Comment:
@johnpierson for the enable marker PR I think something like this is needed for NodeAutoCompleteMarkerVisible